frontend/src/components/Button/index.tsx

```diff
@@ -1,15 +1,26 @@
 import React from 'react';
+import { variantStyles, defaultStyles, sizesStyles } from './theme';
```

Comment:
In my opinion hiding the tailwind classes like this is actually a bad idea. I'd personally inline them all into the classname property with eg. the `classnames` library.
```
className={classnames(
  'disabled:opacity-50', 'disabled:cursor-not-allowed', 'transition-all', 'font-bold', 'font-sans', 'text-sm',
   { 'bg-primary-light hover:bg-primary-main': color == 'primary',
     'bg-error-light': 'color: == 'error'
     (...et cetera)
 }
)}
``